style.css

```diff
@@ -1,6 +1,22 @@
 body {
 	text-align: center;
-	background: #aaf;
+	background-image: url("ship_image_resize.jpg");
+	background-repeat: no-repeat;
+}
+
+h1{
+	text-align: left;
+	color: silver;
+
+	font-size: 30pt;
+	font-family: "Copperplate Gothic";
```

Comment:
As an FYI, you could have also kept both. If you list multiple fonts with a comma, the browser will use the first one it recognizes.